isaric/parsers/brazil-followup.toml

```diff
@@ -0,0 +1,829 @@
+    fields = [
+      { field = "flw_hosp_oxy_new", ref = "Y/N" },
+      { field = "flw_hosp_inva_new", ref = "Y/N" },
+    ]
+
+  [visit.treatment_invasive_ventilation]
+    field = "flw_hosp_inva_new"
+    ref = "Y/N"
+
+    # [visit.treatment_antivirals]
+    #   field = "flw_antiv"
+    #   ref = "Y/N"
+    #   if = { flw_treat_acute = '1' }
+
+    # [visit.treatment_antiviral_type]
+    #   combinedType = "set"
+    #   excludeWhen = "none"
+    #   fields = [
+    #     { field = "flw_antiv_agents___1", values = { 1 = "Lopinavir / Ritonavir" } },
+    #     { field = "flw_antiv_agents___2", values = { 1 = "Darunavir" } },
+    #     { field = "flw_antiv_agents___3", values = { 1 = "Remdesivir" } },
+    #     { field = "flw_antiv_agents___4", values = { 1 = "Favipiravir" } },
+    #     { field = "flw_antiv_agents___5", values = { 1 = "Aciclovir / Ganciclovir" } },
+    #     { field = "flw_antiv_agents___6", values = { 1 = "Oseltamivir" } },
+    #     { field = "flw_antiv_agents___7", values = { 1 = "Other" } },
+    #   ]
+    #   if = { flw_treat_acute = '1' }
+
+    # [visit.treatment_antiviral_type_other]
+    #   combinedType = "set"
+    #   excludeWhen = "none"
+    #   fields = [{ field = "flw_antiv_other" }]
+    #   if = { flw_treat_acute = '1' }
+
+    # [visit.treatment_antibiotics]
+    #   field = "flw_antib"
+    #   ref = "Y/N"
+    #   if = { flw_treat_acute = '1' }
+
+    # [visit.treatment_antibiotics_type]
+    #   combinedType = "set"
+    #   excludeWhen = "none"
+    #   fields = [{ field = "flw_antiv_ag_3" }]
+    #   if = { flw_treat_acute = '1' }
+
+    # [visit.treatment_corticosteroid_type]
+    #   combinedType = "set"
+    #   excludeWhen = "none"
+    #   fields = [
+    #     { field = "flw_wh_ster___1", values = { 1 = "Dexamethasone" } },
+    #     { field = "flw_wh_ster___2", values = { 1 = "Hydrocortisone" } },
+    #     { field = "flw_wh_ster___3", values = { 1 = "Prednisolone" } },
+    #     { field = "flw_wh_ster___4", values = { 1 = "Methylprednisolone" } },
+    #     { field = "flw_wh_ster___5", values = { 1 = "Other" } },
+    #   ]
+    #   if = { flw_treat_acute = '1' }
+
+    # [visit.treatment_other]
+    #   combinedType = "set"
+    #   excludeWhen = "none"
+    #   fields = [{ field = "flw_med_other_name" }]
+    #   if = { flw_treat_acute = '1' }
+
+  [visit.date_outcome]
+    field = "flw_first_disch"
+    description = "What date were you discharged from hospital for the first time?"
+
+  [visit.outcome]
+    field = "flw_first_disch"
+    apply = { function = "textIfNotNull", params = ["discharged"] }
+
+    # OBSERVATION
+    # required fields - phase, date, name
+
+[[observation]]
+  name = "abdominal_pain"
+  start_date = { field = "flw_first_sym" }
+  date = { field = "flw_first_sym", apply = { function = "endDate", params = [
+    14,
+  ] } }
+  duration_type = "event"
+  phase = "initial_illness"
+  is_present = { field = "flw_first_14d___12", ref = "checkbox" }
```

Comment:
i think this is ok re. the phase - so we will have phase = initial_illness for these symptoms reported at the initial illness onset in the freestanding surveys, preadmission (eventually, for pre-admission medications/vaccinations), admission, study and followup. Should we review the South Africa parser for symptom data reported at initial illness also to be consistent? @abhidg what do you think?